app/ide-desktop/lib/dashboard/src/components/UnstyledButton.tsx

```diff
@@ -42,6 +43,15 @@ function UnstyledButton(props: UnstyledButtonProps, ref: React.ForwardedRef<HTML
       </aria.Button>
     </FocusRing>
   )
+
+  return buttonProps['aria-label'] == null ? (
+    button
+  ) : (
+    <aria.TooltipTrigger>
```

Comment:
Please import `TooltipTrigger` from `ariaComponents` as well, because we might want to add extra functionality to `TooltipTrigger` in the future and want to make sure it propagates to all places...